### ISSUE TITLE
-k parameter not working on Ubuntu

### ISSUE THREAD

**Issue by VikramMoule** [sripathikrishnan/redis-rdb-tools#99]
When I use the command `rdb -c json -k 'cluster_*' redis.rdb`

I always get following error :

```
Traceback (most recent call last):
  File "/usr/local/bin/rdb", line 9, in <module>
    load_entry_point('rdbtools==0.1.10', 'console_scripts', 'rdb')()
  File "/usr/local/lib/python2.7/dist-packages/rdbtools/cli/rdb.py", line 86, in main
    parser.parse(dump_file)
  File "/usr/local/lib/python2.7/dist-packages/rdbtools/parser.py", line 326, in parse
    self.parse_fd(open(filename, "rb"))
  File "/usr/local/lib/python2.7/dist-packages/rdbtools/parser.py", line 381, in parse_fd
    self.skip_object(f, data_type)
  File "/usr/local/lib/python2.7/dist-packages/rdbtools/parser.py", line 550, in skip_object
    skip_string(f)
NameError: global name 'skip_string' is not defined
```

If the -k parameter is removed it gives me all keys which is not what I want.

root@localhost.local# python -V
Python 2.7.6
root@localhost.local# pip install rdbtools
Requirement already satisfied (use --upgrade to upgrade): rdbtools in /usr/local/lib/python2.7/dist-packages
Requirement already satisfied (use --upgrade to upgrade): redis in /usr/local/lib/python2.7/dist-packages (from rdbtools)
Cleaning up...

**Comment by amotzg** [comment]
@VikramMoule Please try using the master code, it should have a fix to this as to #96 .

@oranagra is there an expected release to include this?

**Comment by VikramMoule** [comment]
@amotzg Thanks. It works as expected with the master code.

**Comment by oranagra** [comment]
i don't think we have any other release coming soon, so the question is if this fix alone justifies a release.

**Comment by aawilson** [comment]
Is there some way to daylight this in lieu of a release? The only way I found this was by searching in the closed issues of this repo after a google search turned up a result that indicated that it might have been an issue (not a result that pointed to the issue--the page of the result didn't actually have the actual content, it was just one of those "github issue mirroring" sites so I got the clue to search here from that). Not exactly a straight line.
